src/encode.rs

```diff
@@ -22,6 +22,9 @@ use ::bitcoin::consensus::encode as btcenc;
 
 use transaction::{Transaction, TxIn, TxOut};
 
+pub use ::bitcoin::consensus::encode::MAX_VEC_SIZE;
```

Comment:
Can we `pub use` this at the top level so that users can import `elements::VarInt`?

Reply:
Fixed this.